modules/db/src/main/scala/gem/dao/ObservationDao.scala

```diff
@@ -30,7 +40,7 @@ object ObservationDao {
        .unique
        .map { case (t, i) =>
          Observation(id, t, i, Nil)
-       }      
+       }
 
   def select(id: Observation.Id): ConnectionIO[Observation[Step[_]]] =
```

Comment:
I normally want to treat a bogus id as an error: if you have your hands on an id it must have come from the database originally so lookups shouldn't fail. But I can see it both ways. There could be two versions of the method I guess.